File: GameLibrary/src/cc/lib/zombicide/ZColor.java
package cc.lib.zombicide;

import cc.lib.game.GColor;

public enum ZColor {
    BLUE(0, 7, GColor.BLUE),
    YELLOW(8, 19, GColor.YELLOW),
    ORANGE(20, 42, GColor.ORANGE),
    RED(43, 43, GColor.RED);

    ZColor(int dangerPts, int maxPts, GColor color) {
        this.dangerPts = dangerPts;
        this.maxPts = maxPts;
        this.color = color;
    }

    public final int dangerPts;
    public final int maxPts;
    public final GColor color;
}

File: GameLibrary/src/cc/lib/zombicide/ZColor.kt
package cc.lib.zombicide

import cc.lib.annotation.Keep
import cc.lib.game.GColor

@Keep
enum class ZColor(@JvmField val dangerPts: Int, @JvmField val maxPts: Int, @JvmField val color: GColor) {
    BLUE(0, 7, GColor.BLUE),
    YELLOW(8, 19, GColor.YELLOW),
    ORANGE(20, 42, GColor.ORANGE),
    RED(43, 43, GColor.RED);
}